connector_search_engine/models/se_binding.py

```diff
@@ -103,14 +105,63 @@ def unlink(self):
             self.env["se.binding.todelete"].sudo().create(todelete_vals_list)
         return super().unlink()
 
-    def jobify_recompute_json(self, force_export=False):
-        description = _("Recompute %s json and check if need update" % self._name)
+    def _get_binding_to_process(self, bindings, batch_size):
+        return bindings[0:batch_size], bindings[batch_size:]  # noqa: E203
+
+    def jobify_recompute_json(self, force_export=False, batch_size=500):
         # The job creation with tracking is very costly. So disable it.
-        for record in self.with_context(tracking_disable=True):
-            record.with_delay(description=description).recompute_json(
+        bindings = self.with_context(tracking_disable=True)
+        while bindings:
+            processing, bindings = self._get_binding_to_process(bindings, batch_size)
+
+            # We check if we are currently handling an exception in order
+            # to change the job description.
+            (current_exception, current_exception_value, __) = sys.exc_info()
+            description = (
+                (
+                    _(
+                        "Batch task of %s for recompute %s json after previous "
+                        "batch task raised %s: %s."
```

Comment:
```suggestion
                        "Batch of %s items for %s. Previous batch raised %s: %s."
```
The fact that we are recomputing json is in the job func name already. Let's keep this msg shorter ;)